tests/compile/test_silu_mul_quant_fusion.py

```diff
@@ -36,7 +36,7 @@ def test_fusion_silu_and_mul_quant(num_tokens, hidden_size):
     # Reshape pass is needed for the fusion pass to work
```

Comment:
You should also remove the `envs.VLLM_TARGET_DEVICE != "cuda"` above - make it enabled for CUDA and ROCm